src/Microsoft.DotNet.ProjectJsonMigration/transforms/AddBoolPropertyTransform.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
The folder `transforms` appears to start with a lower case `t`.  Can it be `Transforms`?